system/jlib/jfile.cpp

```diff
@@ -470,8 +472,6 @@ bool CFile::setTime(const CDateTime * createTime, const CDateTime * modifiedTime
     if (modifiedTime)
         am.modtime  = timetFromIDateTime (modifiedTime);
     return (utime(filename, &am)==0);
-#else
```

Comment:
NOTE: This is where the issue was -- was failing the ECL compile because this was UNIMPLEMENTED for APPLE